src/lib/store.js

```diff
@@ -1,14 +1,20 @@
 import { createStore, combineReducers } from 'redux';
 import { createActions, handleActions } from 'redux-actions';
-import undoable, { ActionCreators } from 'redux-undo';
+import undoable, { ActionCreators, ActionTypes } from 'redux-undo';
 import {
   normalizeTheme,
   normalizeThemeColor,
   normalizeThemeColors
 } from './utils';
 
 // Actions that should trigger a theme update in URL history and the add-on
-export const themeChangeActions = ['SET_THEME', 'SET_COLOR', 'SET_BACKGROUND'];
+export const themeChangeActions = [
+  'SET_THEME',
```

Comment:
A nit: since it's a string with meaningful content, I'd prefer to see this as a constant.